src/run.jl

```diff
@@ -43,11 +62,107 @@ julia> sol.t[end]
 0.1
 ```
 """
-function trixi_include(mod::Module, parameters_file::AbstractString; kwargs...)
-  Base.include(ex -> replace_assignments(ex; kwargs...), mod, parameters_file)
+function trixi_include(mod::Module, elixir::AbstractString; kwargs...)
+  Base.include(ex -> replace_assignments(ex; kwargs...), mod, elixir)
 end
 
-trixi_include(parameters_file::AbstractString; kwargs...) = trixi_include(Main, parameters_file; kwargs...)
+trixi_include(elixir::AbstractString; kwargs...) = trixi_include(Main, elixir; kwargs...)
+
+
+"""
+    convergence_test(elixir::AbstractString, iterations; kwargs...)
+
+Run multiple Trixi simulations with the parameters given in `elixir` and compute
+the experimental order of convergence (EOC) in the ``L^2`` and ``L^\\infty`` norm.
+The number of runs is specified by `iterations`; in each run the initial
+refinement level will be increased by 1. Parameters can be overriden by specifying them as
```

Comment:
Maybe more like this?
```suggestion
refinement level will be increased by 1. Parameters, i.e., variables that are assigned to, can be overriden by specifying them as
```

Reply:
But maybe it would also be sufficient to mention `trixi_include` in `See also:...`

Reply:
Also, it should be mentioned that `initial_refinement_level` must exist as an assignment somewhere (as a variable or as a kwargs assignment) in the elixir.

Reply:
The constructor for `TreeMesh` we use in all elixirs has `initial_refinement_level` as required keyword argument.